Gemfile

```diff
@@ -5,6 +5,7 @@ gem 'rails', '~> 4.2.6'
 gem 'rails-api'
 gem 'active_model_serializers', '~> 0.10.0'
 gem 'rack-cors', require: 'rack/cors'
+gem 'rb-readline'
```

Comment:
I don't believe this is correct.  The latest template does not include rb-readline.